main.py

```diff
@@ -14,6 +14,7 @@
 import github_api
 import git_commands
 import aider_coder
+from celery_tasks import task_create_pull_request_for_issue, task_handle_pr_review_comment, task_handle_issue_comment
```

Comment:
@Aiderbot I'm getting a circular import error because the celery_tasks file is importing from main, and main is importing from celery_tasks. So I think we should move the functions from main.py into celery_tasks.py, and just use main.py for the webhook handler.